src/Microsoft.VisualBasic/tests/Microsoft.VisualBasic.Tests.csproj

```diff
@@ -23,6 +23,7 @@
     <Compile Include="Microsoft/VisualBasic/CompilerServices/StandardModuleAttributeTests.cs" />
     <Compile Include="Microsoft/VisualBasic/Devices/NetworkAvailableEventArgsTests.cs" />
     <Compile Include="ConversionsTests.cs" />
+    <Compile Include="Microsoft\VisualBasic\CompilerServices\StructUtilsTests.cs" />
```

Comment:
Could you please put in alpha order? Feel free to change all the slashes to backslashes for consistency.

Reply:
Fixed.